src/client/app/components/maps/MapCalibrationInitiateComponent.tsx

```diff
@@ -90,13 +98,19 @@ class MapCalibrationInitiateComponent extends React.Component<MapInitiatePropsWi
 			</form>
 		);
 	}
-
 	private async confirmUpload(event: any) {
 		const bcheck = this.handleAngle(event);
 		if (bcheck) {
-			await this.handleInput(event);
-			await this.notifyLoadComplete();
-			this.props.updateMapMode(CalibrationModeTypes.calibrate);
+			if (this.fileInput.current.files.length === 0) {
+				this.notifyBadMapLoad();
+			}
+			else if (this.state.mapName.trim() === '')	{
+				this.notifyBadName();
+			}
+			else {
+				await this.handleInput(event);
+				this.props.updateMapMode(CalibrationModeTypes.calibrate);
```

Comment:
I notice the notifyLoadComplete() was removed on a successful check situation. I have wondered about it and my testing indicates that it does not really tell you if a bad map is uploaded. Thus, I think it is okay to remove it but the function that was being called and related code should be removed if we no longer do this.